src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -227,6 +227,10 @@ private CommandResult executeCommand(String commandText) throws CommandException
                 handleQuestionnaire();
             }
 
+            if (commandResult.isShowCalendar()) {
```

Comment:
I think u forgot to add the `calendarWindow.hide();` in the handleExit() method

Reply:
Thanks! fixed!